planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/OptimizerInitialDeployment.java

```diff
@@ -108,9 +108,9 @@ private String openBenchmarkOffersDefaultFile() {
          int numPlansToGenerate, double hyst) {
 
       log.debug("Optimization method started. Inputs received");
-      log.debug("AAM is: " + appModel);
-      log.debug("Suitable offers are: " + suitableCloudOffer);
-      log.debug("Informaton of Benchmark platform is: " + benchmarkPlatformsYaml);
+      log.debug("AAM is: {}", appModel);
```

Comment:
- try to avoid multi-lines logs. 
- `log.info` is more [appropriate](http://stackoverflow.com/questions/6860186/log-info-vs-log-debug)